README.md

```diff
@@ -30,6 +30,8 @@ how to install Howl from source.
 based system).
 - `C compiler`: Howl has a very small C core itself, and it embeds
 dependencies written in C.
+- If you're using Windows, you *need* to build Howl inside of
```

Comment:
Should it be in a readme (e.g. `README.windows.md`), or do you think it would be better as a page on the site (e.g. `getit.windows.html`), as a parallel to `getit.html`?